tests/test_conditional_steps.py

```diff
@@ -25,16 +27,34 @@ def check_conditional_step_success(tmpdir, capsys, conditional_step_passed):
 def build_config_file(tmpdir, conditional_step_passed):
     conditional_step_exit_code = 0 if conditional_step_passed else 1
 
+    true_branch_step = Step(
+        name=true_branch_step_name,
+        command=['touch', true_branch_step_name],
+        artifacts=true_branch_step_name)
+
+    false_branch_step = Step(
+        name=false_branch_step_name,
+        command=['touch', false_branch_step_name],
+        artifacts=false_branch_step_name)
+
+    conditional_step = Step(
+        name=conditional_step_name,
+        command=['bash', '-c', f'touch {conditional_step_name}; exit {conditional_step_exit_code}'],
+        artifacts=conditional_step_name)
+
     config = inspect.cleandoc(f'''
         from universum.configuration_support import Configuration, Step
 
-        true_branch_step = Step(name='{true_branch_step_name}', command=['touch', '{true_branch_step_name}'])
-        false_branch_step = Step(name='{false_branch_step_name}', command=['touch', '{false_branch_step_name}'])
-        conditional_step = Configuration([dict(name='conditional',
-            command=['bash', '-c', 'exit {conditional_step_exit_code}'],
-            if_succeeded=true_branch_step, if_failed=false_branch_step)])
+        true_branch_step = Step(**{str(true_branch_step)})
```

Comment:
It looks like this variable is not needed. But I think you can keep it for readability.